MvvmCross-Forms/MvvmCross.Forms/Views/MvxFormsExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace MvvmCross.Forms.Views
+{
+    public static class MvxFormsExtensions
+    {
+        public static bool PageMatchesViewModel(this Page page, Type viewModelType)
```

Comment:
I would like to get this naming more into the .NET conventions. Maybe something like `IsViewModelTypeOf(viewmodelType)`